strapi/strapi
# fix i18n wrapParams ignoring lower services

### What does it do?

Changed wrapParams to use the params it gets from the lower services so that they are not ignored

### Why is it needed?

that no other entity service decorators laying lower in the structure could use wrapParams since there output was just ignored

### How to test it?

use a plugin that uses wrapParams function on the entity service and see that the wrapParms information is now being used
